sources/packages/backend/libs/services/src/queue/bull-board-queues-module.ts

```diff
@@ -0,0 +1,45 @@
+import { Inject, LoggerService, Module, OnModuleInit } from "@nestjs/common";
+import { ModuleRef } from "@nestjs/core";
+import { BULL_BOARD_INSTANCE, BullBoardInstance } from "@bull-board/nestjs";
+import { QueueService } from "@sims/services/queue/queue.service";
+import { InjectLogger } from "@sims/utilities/logger";
+import { getQueueToken } from "@nestjs/bull";
+import { Queue } from "bull";
+import { BullAdapter } from "@bull-board/api/bullAdapter";
+
+@Module({})
+export class BullBoardQueuesModule implements OnModuleInit {
+  constructor(
+    private readonly moduleRef: ModuleRef,
+    private readonly queueService: QueueService,
+    @Inject(BULL_BOARD_INSTANCE)
+    private readonly board: BullBoardInstance,
+  ) {}
+
+  /**
+   * Adds all queues to the bull board during application initialization.
+   * checking if the queue is active and if it is a scheduler.
+   */
```

Comment:
Minor please check the comment and period.

Also please add returns.

Reply:
Add the period.
About the return type, I do not believe that we usually add comments for Promise<void>. Please let me know if that is what you meant.

Reply:
You are right about the return type.